src/main/java/hu/wurfel/new_school_reference/diarySubjectTeacherStudent/DiarySubjectTeacherStudentController.java

```diff
@@ -0,0 +1,34 @@
+package hu.wurfel.new_school_reference.diarySubjectTeacherStudent;
+
+import org.modelmapper.ModelMapper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("/api/v1/diary_subject_teacher_student")
+public class DiarySubjectTeacherStudentController {
```

Comment:
@kafnyi 
Most, hogy jobban belegondolok, ennek a kontrollernek a létezése alapvetően eléggé kérdéses. Szerintem ennek nem kell controller, mert egy csak egy kötő object. Ezt nem biztos, de amennyire most látom a rendszer (és ez nem teljes), ennek a service inkább legyen behúzva azokba a service-ekbe, ahova kapcsolódok és kell. Ha fakme és nem akarod átírni, akkor Írd fel ezt is és pénteken átbeszéljük.

Reply:
pénteken átbeszélhetjuk, mert ennek pont látom létjogosultségát.

Reply:
@kafnyi 
ezt nem beszéltük meg, úgy hogy írgyad le 😄